recipes/plio/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "plio" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a189e7ee6973a209aee81ca412144599fdf7d2aefbb03c1d93c37212be053a7b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pvl
+    - protobuf
+    - gdal
+    - icu
+    - h5py
+    - pandas
+    - sqlalchemy
+    - pyyaml
+    - affine
+    - networkx
+    - scipy
+    - certifi
+    - pyproj
+    - jinja2
+  run:
+    - python
+    - setuptools
+    - numpy
+    - pvl
+    - protobuf
+    - gdal
+    - icu
+    - h5py
+    - pandas
+    - sqlalchemy
+    - pyyaml
+    - affine
+    - networkx
+    - scipy
+    - certifi
+    - pyproj
+    - jinja2
+
+test:
+  imports:
+    - plio
```

Comment:
As a fellow astro person, the overlap with FITSIO PLIO image compression here is unfortunate! ;p

Reply:
I had not idea! It is amazing the format separation between astronomy and surface planetary science. Maybe the new geoFITS will help some with that?

Reply:
Hahahah. Maybe!